lib/python/qmk/cli/ci/validate_aliases.py

```diff
@@ -0,0 +1,32 @@
+"""Validates the list of keyboard aliases.
+"""
+from pathlib import Path
+
+from milc import cli
+
+from qmk.constants import QMK_FIRMWARE
+from qmk.json_schema import json_load
+from qmk.keyboard import resolve_keyboard, keyboard_folder
+
+
+def _safe_keyboard_folder(target):
+    try:
+        return keyboard_folder(target)
+    except Exception:
+        return None
+
+
+@cli.subcommand('Validates the list of keyboard aliases.', hidden=True)
+def ci_validate_aliases(cli):
+    aliases = json_load(Path('data/mappings/keyboard_aliases.hjson'))
+
+    success = True
+    for alias in aliases.keys():
+        if alias == 'bm16a':
+            pass
```

Comment:
Whats the story on this exception?

Reply:
Shouldn't be one -- this was me adding a piece of code I could set a breakpoint for. Will remove.